package.json

```diff
@@ -19,8 +19,8 @@
         "fetch:stats": "node tools/fetch-stats.js",
         "fetch:sponsors": "node tools/fetch-sponsors.js",
         "fetch": "npm-run-all --parallel fetch:*",
-        "start": "npm-run-all build:sass --parallel watch:*  --parallel images",
-        "build": "npm-run-all build:sass --parallel build:*   --parallel images"
+        "start": "npm-run-all --parallel watch:* images",
+        "build": "npm-run-all --parallel build:* images"
```

Comment:
I’m pretty sure we are intentionally running build:sass first and then the rest in parallel

Reply:
But? `build:sass` is run first and then again `watch:sass` is run in parallel again since we use `watch:*`?

Reply:
Also in the `build` command the `sass` is built first and then again it is being built in parallel? when using `build:*`

Reply:
For watch: I think this is because we need an initial sass build for the other builds to work, then watch:sass watches for changes.

For build: I think we still need sass built first and it’s run a second time as a consequence of the wildcard rather than intentional.

Reply:
`other builds to work` but new.eslint.org doesn't pull any other bundles right? we won't need `build:sass`. even playground doesn't have this? Also how ever watch command is going to do a initial build (generates css) and then watch changes? 

- https://github.com/eslint/playground/blob/7a052c4fcbdd97e866fbf2bbe3f3e8e52289cd9a/tools/dev-server.js#L153-L155

For build: I've fixed it to run sass first and then eleventy

Reply:
What I mean is we need to build sass before we build Eleventy. The CSS files get placed into assets/css, so then Eleventy can pick those up when the site is generated.

The same is true for watching. Otherwise, we don’t know which task is run first, so the CSS files may not be in place when Eleventy is first built.

Reply:
I've added the `build:sass` to the start command and remove unwanted parallel command + stopped running `build:sass` twice